plans/video-generation.md

```
@@ -0,0 +1,1140 @@
+实现视频生成模式支持，作为 Polling Agent 下的一个子 Agent，支持多种视频生成模型。
+
+---
+
+## 1. 核心架构设计
+
+### 1.1 层级结构
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)